src/backend/distributed/planner/recursive_planning.c

```diff
@@ -1151,7 +1151,7 @@ RecursivelyPlanSubquery(Query *subquery, RecursivePlanningContext *planningConte
 	 */
 	resultQuery = BuildSubPlanResultQuery(subquery->targetList, NIL, resultId);
 
-	if (log_min_messages <= DEBUG1 || client_min_messages <= DEBUG1)
+	if (IsLoggableLevel(DEBUG1))
```

Comment:
To me this change is not an improvement `IsLoggableLevel` doesn't reveal much about what we're trying to do. When I see `log_min_messages ` / `client_min_messages ` I can understand what we're trying to do, but with `IsLoggableLevel ` not.

Besides, Postgres mostly follows the convention before the change, I'd prefer to stick to that

Reply:
Why do you think it doesnt reveal what we are trying to do? When I ask the question `what does this if check do?` the answer will probably be: `It checks if DEBUG1 is a level that we can log, so basically it answers the question: `isLoggableLevel`.

Reply:
> It checks if DEBUG1 is a level that we can log

In postgres, `log` can be configured differently for [clients](https://www.postgresql.org/docs/9.5/runtime-config-client.html#GUC-CLIENT-MIN-MESSAGES) like psql, and for [server logs](https://www.postgresql.org/docs/9.5/runtime-config-logging.html#GUC-LOG-MIN-MESSAGES).  When you use `log` in the function name, you're slightly hiding `client_min_messages` from this context. So, the term `loggable` is not as clear to me as you think.

Also, for such a simple function, do we really need source/header file(s)? What else are we planning to put there in the future?

I'm not hardly against merging this to master, what I'm just saying is that I couldn't find this change as an improvement over what we have, so I'm neutral, I'm not planning to approve. If someone else thinks that it is a nice improvement, let's go ahead and merge.